apps/charterafrica/src/components/Entity/Entity.js

```diff
@@ -43,12 +50,24 @@ const SocialMediaLink = React.forwardRef(function SocialMediaLink(props, ref) {
     slack: SlackIcon,
     linkedin: LinkedInIcon,
     telegram: TelegramIcon,
+    discord: DiscordIcon,
+    tiktok: TikTokIcon,
+    whatsapp: WhatsAppIcon,
+    instagram: InstagramIcon,
+    youtube: YouTubeIcon,
   };
   return href && icons[variant] ? (
-    <Link ref={ref} href={href}>
+    <Link
+      ref={ref}
+      href={href}
+      sx={{
+        padding: 0,
+      }}
+    >
       <SvgIcon
         inheritViewBox
         component={icons[variant]}
+        viewBox="0 0 24 24"
```

Comment:
```suggestion
```

`"0 0 24 24"` is the default `viewBox`. Only set if different e.g. when it's "0 0 25 25".

Reply:
@kilemensi I'm kind of lost in the implementation here. The only way to know if the viewbox is not "0 0 24 24" is by actually looking into the SVG itself. Not sure how we can accomplish that.

Reply:
The SVG are not automatically generated @kelvinkipruto. For each SVG, we should know the exact size/viewBox.